app/Schema/Mysql.php

```diff
@@ -1069,17 +1069,32 @@ function version_46(PDO $pdo)
     $pdo->exec("CREATE UNIQUE INDEX task_has_links_unique ON task_has_links(link_id, task_id, opposite_task_id)");
 
     $rq = $pdo->prepare('INSERT INTO links (label, opposite_id) VALUES (?, ?)');
+    
+    # ID cannot be known at time of record creation so we have to update it after the fact
+    # On MariaDB clusters auto-increment size is normally != 1, so relying on increments of 1 would break
+    $arq = $pdo->prepare('UPDATE links SET opposite_id=? WHERE label=?');
+
     $rq->execute(array('relates to', 0));
-    $rq->execute(array('blocks', 3));
-    $rq->execute(array('is blocked by', 2));
-    $rq->execute(array('duplicates', 5));
-    $rq->execute(array('is duplicated by', 4));
-    $rq->execute(array('is a child of', 7));
-    $rq->execute(array('is a parent of', 6));
-    $rq->execute(array('targets milestone', 9));
-    $rq->execute(array('is a milestone of', 8));
-    $rq->execute(array('fixes', 11));
-    $rq->execute(array('is fixed by', 10));
+
+    $rq->execute(array('blocks', 0));
+    $rq->execute(array('is blocked by', get_last_insert_id($pdo)));
```

Comment:
From where is coming the function `get_last_insert_id()`?

Reply:
I've seen it used elsewhere: https://github.com/kanboard/kanboard/blob/master/app/Schema/Migration.php#L70

Reply:
Ah ok. I don't even remember what I did :)

Give me some time to test your pull-request on a Galera cluster.

Reply:
Great, thanks a lot! We tested it on an MMM cluster, but Galera is better, it automatically changes the `auto_increment_increment` as nodes join and leave the cluster.